core/src/test/java/org/elasticsearch/index/query/SpanNotQueryBuilderTest.java

```diff
@@ -66,14 +66,22 @@ public void testValidate() {
         int totalExpectedErrors = 0;
         SpanQueryBuilder include;
         if (randomBoolean()) {
-            include = new SpanTermQueryBuilder("", "test");
+            if (randomBoolean()) {
+                include = new SpanTermQueryBuilder("", "test");
+            } else {
+                include = null;
```

Comment:
same as above, we can omit setting explicitly to null? or maybe we want to still randomly do it?